### DISCUSSION TITLE
Any idea if and when gzip sources can be added manifest instllers?

### DISCUSSION THREAD

**Discussion by nunorelvao** [microsoft/winget-cli#4900] [Ideas]
As an example "https://github.com/derailed/popeye/releases" only has **gzip** format sources ,would be nice to have it, and latest [spec 
#140](https://github.com/microsoft/winget-cli/blob/master/doc/specs/%23140%20-%20ZIP%20Support.md) from 2022 does reffer this can be added in future. **When is the future?**

**Comment by Trenly** [comment]
There is not a specified date yet, but here is the tracking issue - https://github.com/microsoft/winget-cli/issues/2899

Issues are prioritized based on several factors, including the number of 👍 they have. Adding your 👍 to the issue's first comment will help it be prioritized sooner

**Comment by pl4nty** [comment]
k9s has .zip files for Windows: https://github.com/derailed/k9s/releases/download/v0.32.5/k9s_Windows_amd64.zip

**Reply by nunorelvao** [reply]
Yes , that one i already contributed directly and have updated winget packages, the other  one i mentioned has gz only
